docs/source/tutorials/chat.rst

```diff
@@ -294,6 +295,7 @@ object.
             # For local csv files, we specify "csv" as the source, just like in
             # load_dataset
             source="csv",
+            split="train",
```

Comment:
so I just tested this example with your config and dataset with `split` removed and it worked. I also kept it and it worked, so I suppose this doesn't affect local files. Since this is typically used for datasets on HF hub, I would suggest removing this to avoid confusion since it is not required for local files.